specs/protocol/system-config.md

```diff
@@ -122,12 +121,6 @@ manner, it is stored at a special storage slot corresponding to
 Unlike the other values, the `unsafeBlockSigner` only operates on blockchain
 policy. It is not a consensus level parameter.
 
-### `validatorRewardScalar` (`uint256`)
```

Comment:
Here also, how about adding deprecated mark and description that it is totally removed after MPT hardfork instead of removing it?

Reply:
Or adding a part about removal of validatorRewardScalar in SystemConfig contract would be nice

Reply:
Agreed with the first part, I'll revive and mark as deprecated, since as I know we are not removing this value from systemConfig contract at the time of MPT upgrade.